pymc3/gp/cov.py

```diff
@@ -88,7 +92,8 @@ def __array_wrap__(self, result):
 class Combination(Covariance):
     def __init__(self, factor_list):
         input_dim = np.max([factor.input_dim for factor in
-                            filter(lambda x: isinstance(x, Covariance), factor_list)])
+                            filter(lambda x: isinstance(x, Covariance),
```

Comment:
Our BDFL (Python's not PyMC3's) generally frowns upon `filter` and `map` these days, as much as it makes my Haskell-loving heart cry a bit.  More Pythonic may be:

```python
[x for x in factor_list if isinstance(x, Covariance)]
```